pkg/kubectl/cmd/replace.go

```diff
@@ -33,7 +33,9 @@ import (
 const (
 	replace_long = `Replace a resource by filename or stdin.
 
-JSON and YAML formats are accepted.`
+JSON and YAML formats are accepted.
+
+Please refer to the models in http://kubernetes.io/v1.0/docs/api-reference/definitions.html to find if a field is mutable.`
```

Comment:
Shouldnt this be linked to HEAD here?
HEAD will be replaced by the appropriate version by versionize-docs.sh